build-ps/debian/extra/apparmor-profile-router

```diff
@@ -29,8 +29,7 @@
   /etc/ssl/openssl.conf r,
 
 # Allow runtime files such as PID-file
-  /var/run/mysqlrouter/** rw,
-  /run/mysqlrouter/** rw,
+  /{,var/}run/mysqlrouter/** rw,
```

Comment:
As above, I'm even not sure if we install AA profile for the router. Probably not.